packages/components/organisms/f-checkout/src/components/_tests/AgeVerification.test.js

```diff
@@ -0,0 +1,231 @@
+                expect(wrapper.vm.months).toMatchSnapshot();
+            });
+        });
+
+        describe('years ::', () => {
+            beforeEach(() => {
+                // Arrange
+                wrapper = shallowMount(AgeVerification, {
+                    i18n,
+                    localVue,
+                    store: createStore()
+                });
+            });
+
+            it('should return an array of 100 year objects', () => {
+                // Assert
+                expect(wrapper.vm.years.length).toEqual(100);
+                expect(wrapper.vm.years).toMatchSnapshot();
+            });
+        });
+    });
+
+    describe('methods ::', () => {
+        describe('`initDateOfBirth` ::', () => {
+            describe('when `customer.dateOfBirth` exists', () => {
+                it('should update `selectedDate` with `dateOfBirth`', () => {
+                    // Arrange
+                    const day = '4';
+                    const month = '7';
+                    const year = '1990';
+
+                    const dateOfBirth = new Date(year, month, day);
+
+                    wrapper = shallowMount(AgeVerification, {
+                        i18n,
+                        localVue,
+                        store: createStore({
+                            ...defaultCheckoutState,
+                            customer: {
+                                dateOfBirth
+                            }
+                        })
+                    });
+
+                    // Act
+                    wrapper.vm.initDateOfBirth();
+
+                    // Assert
+                    expect(wrapper.vm.selectedDate.day).toEqual(day);
+                    expect(wrapper.vm.selectedDate.month).toEqual(month);
+                    expect(wrapper.vm.selectedDate.year).toEqual(year);
+                });
+            });
+
+            describe('when `customer.dateOfBirth` does not exist', () => {
+                const currentYear = '2025';
+
+                beforeEach(() => {
+                    // Arrange
+                    jest.spyOn(Date.prototype, 'getUTCFullYear').mockImplementation(() => currentYear);
+
+                    wrapper = shallowMount(AgeVerification, {
+                        i18n,
+                        localVue,
+                        store: createStore({
+                            ...defaultCheckoutState,
+                            customer: {
+                                dateOfBirth: null
+                            }
+                        })
+                    });
+
+                    // Act
+                    wrapper.vm.initDateOfBirth();
+                });
+
+                it('should update `selectedDate.day` with "1"', () => {
+                    // Assert
+                    expect(wrapper.vm.selectedDate.day).toEqual('1');
+                });
+
+                it('should update `selectedDate.month` with "0"', () => {
```

Comment:
Hard to know without context, but why does this get set to `0`? Does this represent no selected value when it's displayed on screen to the user? If so, may be worth adding a bit more description to the test title (especially as the day gets set to `1`?)

Reply:
Thanks, I'll add some description. Annoyingly I'm assuming dates are stored as an array, so when you do `getMonth()` January = 0, Feb =1 etc.

Reply:
@ashleynolan I've updated the test description and added a comment, Is that any better?

Reply:
Looks good 👍